app/views/disbursements/_select_organization.html.erb

```diff
@@ -0,0 +1,42 @@
+<%# locals: (form:, field_name:, disabled:, events:, default_event:, hide_balances: false, sending: false, receiving: false) %>
+
+<div class="field" data-controller="organization-select">
+    <%= form.label field_name, sending ? "From" : "To" %>
+    <select name="disbursement[<%= field_name %>]" id="disbursement_<%= field_name %>" <%= "disabled" if disabled %> class="bg-[transparent] dark:bg-darkless input input--select flex items-center select-none cursor-pointer disabled:cursor-default" data-organization-select-target="dropdown">
+        <option value>Select one...</option>
+        <option <%= "default" if default_event.present? %> value="<%= default_event.public_id if default_event.present? %>"><%= default_event.name if default_event.present? %></option>
+    </select>
+
+    <div data-organization-select-target="wrapper" style="max-width: 384px;">
+        <input data-organization-select-target="search" style="display: none; border-bottom-left-radius: 0px; border-bottom-right-radius: 0px;" type="text" placeholder="Search">
+
+        <div data-organization-select-target="menu" style="display: none; border-width: 1px; border-style: solid; border-top: none; max-width: 384px; border-top-left-radius: 0px; border-top-right-radius: 0px;" class="border-smoke rounded-md dark:border-black overflow-hidden h-40">
+            <div class="w-full h-full rounded-md overflow-y-scroll" style="border-top-left-radius: 0px; border-top-right-radius: 0px; content-visibility: auto;">
+                <% events.each_with_index.map do |event, i| %>
+                    <% disabled_message = nil %>
+                    <% disabled_message = "Insufficient balance" if sending && !current_user.admin? && event.balance_available <= 0 %>
+                    <% disabled_message = "HCB transfers disabled" if sending && event != default_event && !policy(event).create_transfer? %>
```

Comment:
How come we don't disable it if `event == default_event`? I think I might be missing something - is it for members to be able to preview the form?

Reply:
People can change the to event, it doesn't always match default_event

Reply:
Yes - but how come if they don't have permission to create the transfer to that event, we don't disable it

Reply:
I'm asking why this line:

```ruby
sending && event != default_event && !policy(event).create_transfer?
```

is not:

```ruby
sending && !policy(event).create_transfer?
```